app/components/decision-link.js

```diff
@@ -0,0 +1,32 @@
+import Component from '@glimmer/component';
+import { trackedTask } from 'ember-resources/util/ember-concurrency';
+import { service } from '@ember/service';
+import { task } from 'ember-concurrency';
+
+export default class DecisionLinkComponent extends Component {
+  @service store;
+
+  fetchPublishedExtractTask = task(async () => {
+    const uittreksels = await this.store.query('uittreksel', {
+      'filter[behandeling-van-agendapunt][besluiten][:uri:]': this.decisionURI,
+      'fields[uittreksels]': 'uri',
+    });
```

Comment:
yeah it would be kinda awkward to get the extracts to line up with the decisions if we fetch them all at the same time, this way is easier and not really a perf hit